library/src/main/java/com/woxthebox/draglistview/swipe/ListSwipeItem.java

```diff
@@ -314,4 +336,18 @@ void handleSwipeMove(float dx, RecyclerView.ViewHolder viewHolder) {
         }
         swipeTranslationByX(dx);
     }
+
+    private enum SwipeState {
```

Comment:
Were these enums moved to bottom because of auto formatting or because of something else?
Same with the move of the setSupportedSwipeDirection() method.

I have all enums in top of the class in this lib/project so it is better to keep following that pattern if you don't have a specific reason to move them.